desktop/test/utils/domain/auctions/urls_test.js

```diff
@@ -0,0 +1,19 @@
+const rewire = require('rewire')
+const urls = rewire('../../../../utils/domain/auctions/urls')
+
+describe('Url Utilities', () => {
+  before(() => {
+    urls.__set__('sd', {PREDICTION_URL: 'http://www.liveauctions.com'})
+  })
+  describe('#liveAuctionUrl builds a url to a live auction', () => {
+    it('appends /login with a true "isLoggedIn" option in the second argument', () => {
+      urls.liveAuctionUrl('foo', { isLoggedIn: Boolean({'currentUserOrNull': 'currentUser'}) }).should.equal('http://www.liveauctions.com/foo/login')
+    })
```

Comment:
Noticed you replaced this above with `PREDICTION_URL`; can you keep this consistent? Also note that its ok to use whatever url as long as your not testing the correctness of the domain.